src/controllers/report_builder_controller.js

```diff
@@ -0,0 +1,167 @@
+import { Controller } from '@hotwired/stimulus';
+import { Turbo } from '@hotwired/turbo-rails';
+
+export default class extends Controller {
+  static targets = ['modelSelect', 'hiddenColumnsConfig', 'selectedColumnsList', 'selectedColumnRow', 'calculatedColumnFrame'];
+  static values = { filterTypesMap: Object };
+
+  modelChanged(event) {
+    const selectedModel = event.target.value;
+    if (selectedModel) {
+      const frame = document.getElementById('report_model_context');
+      frame.src = `/custom_reports/model_context?report_model=${encodeURIComponent(selectedModel)}`;
+    }
+  }
+
+  filterColumnChanged(event) {
```

Comment:
@calebtocco is this used anywhere? I might be just missing it, been review the code this weekend